modules/standard/CommDiagnostics.chpl

```diff
@@ -351,9 +351,9 @@ module CommDiagnostics
       D(loc.id).test_nb = chpl_numCommTestNB();
       D(loc.id).wait_nb = chpl_numCommWaitNB();
       D(loc.id).try_nb = chpl_numCommTryNB();
-      D(loc.id).fork = chpl_numCommForks();
-      D(loc.id).fork_fast = chpl_numCommFastForks();
-      D(loc.id).fork_nb = chpl_numCommNBForks();
+      D(loc.id).execute_on = chpl_numCommForks();
```

Comment:
I think that I can remove these methods and that extern record support is good enough that the module code could just use chpl_getCommDiagnosticsHere. Do you think that would be reasonable to do in this PR?